R/dataset-list.R

```diff
@@ -14,7 +14,7 @@ datasets <- c(
                                                               by = c("date")][, region := "United Kingdom"]
                cases <- data.table::rbindlist(list(cases, uk_cases),
                                                fill = TRUE, use.names = TRUE)},
-             stan_args = list(nhsregions = TRUE)),
+             data_args = list(nhsregions = TRUE)),
   Region$new(name = "united-kingdom",
```

Comment:
names should be unique - use folder_name and dataset_folder_name to locate the folder.

e.g.
name = united-kingdom-admissions, folder_name = united-kingdom, dataset_folder_name = admissions